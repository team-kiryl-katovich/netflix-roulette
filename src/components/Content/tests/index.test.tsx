import * as React from 'react';
import * as enzyme from 'enzyme';

import { Content } from '..';

describe('[COMPONENTS] [CONTENT]', () => {
  test('should render component', () => {
    const component = enzyme.shallow(<Content />);
    expect(component).toMatchSnapshot();
  });
});
