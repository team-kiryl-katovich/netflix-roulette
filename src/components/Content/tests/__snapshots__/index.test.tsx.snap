// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENTS] [CONTENT] should render component 1`] = `
<div
  className="content"
>
  <Connect(Component) />
</div>
`;
